jbehave-support-core/src/main/java/org/jbehavesupport/core/internal/web/action/FocusWebAction.java

```diff
@@ -0,0 +1,21 @@
+package org.jbehavesupport.core.internal.web.action;
+
+import org.jbehavesupport.core.web.WebActionContext;
+import org.openqa.selenium.WebElement;
+import org.springframework.stereotype.Component;
+
+@Component
+public class FocusWebAction extends AbstractWebAction {
+
+    @Override
+    public String name() {
+        return "FOCUS";
+    }
+
+    @Override
+    public void perform(WebActionContext ctx) {
+        WebElement element = findElement(ctx);
+
+        scrollIntoView(ctx, element);
```

Comment:
AFAIK "scroll into view" is not the same as "focus", so either is this action badly named or implemented

Reply:
is it possible to add test for it ?

Reply:
If I remember correctly scrollIntoView uses JavaScript - are we able to do implementation without it to support more webdrivers?

Reply:
I rewrited action to use Selenium and renamed it.
But I wasn't able to test it, so to test it I added web property to verify is element is displayed on screen. And it is using javascript again :(